rule.go

```diff
@@ -419,6 +419,21 @@ func (p PCRE) String() string {
 	return s.String()
 }
 
+//string returns a string for all of the flowbits
```

Comment:
All comments need a space following `//` shoudl start with a capital letter and end with a `.` See Effective Go[0] section on Commentary.

// String returns a string for a flowbit.

[0] https://golang.org/doc/effective_go.html#commentary